model/jpa/src/main/java/org/keycloak/organization/jpa/OrganizationAdapter.java

```diff
@@ -93,6 +94,25 @@ public String getName() {
         return entity.getName();
     }
 
+    @Override
+    public String getAlias() {
+        return entity.getAlias();
+    }
+
+    @Override
+    public void setAlias(String alias) {
+        if (entity.getAlias() != null && !entity.getAlias().equals(alias)) {
+            throw new ModelValidationException("Cannot change the alias");
+        }
+        if (StringUtil.isBlank(alias)) {
+            alias = getName();
```

Comment:
Defaults to the `name` if not set. As mentioned in the description, the only reason for that is to make it easier to create an organization if you don't care set an alias and the name is OK for you.

Using the name as an alias might have implications when referencing an organization in places where you eventually need to deal with special characters, spaces, etc ...